cypress/integration/tests/controls/tags/tag/crud.test.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference types="cypress" />
+
+import {
+    login,
+    exists,
+    expandRowDetails,
+    existsWithinRow,
+    closeRowDetails,
+    notExistsWithinRow,
+} from "../../../../../utils/utils";
+import { Tag } from "../../../../models/tags";
+
+import { tdTag } from "../../../../types/constants";
+import * as data from "../../../../../utils/data_utils";
+
+describe("Tag CRUD operations", () => {
+    beforeEach("Login", function () {
+        // Perform login
+        login();
+
+        // Interceptors
+        cy.intercept("POST", "/api/controls/tag*").as("postTag");
+        cy.intercept("GET", "/api/controls/tag*").as("getTag");
+        cy.intercept("PUT", "/api/controls/tag/*").as("putTag");
+        cy.intercept("DELETE", "/api/controls/tag/*").as("deleteTag");
+    });
+
+    it("Tag CRUD", function () {
+        // Create new tag
+        const tag = new Tag(data.getRandomWord(6), data.getExistingTagtype());
+        tag.create();
+        cy.wait("@postTag");
+
+        // Assert that created tag exists
+        expandRowDetails(tag.tagtype);
+        existsWithinRow(tag.tagtype, tdTag, tag.name);
+        closeRowDetails(tag.tagtype);
```

Comment:
- If `tdTag` is going to be constant then we can keep it as default argument in method - existsWithinRow
- Also we can change method name to `existsTag` and `notExistsTag`
- Also as these methods are specific to tag; can we move it in tag model file?
 @nitishSr what is your view?

Reply:
This method should work generally, I didn't try, but made it by keeping in view that, we should be able to use it to see and read the data, within expanded view. Like the stakeholder members in stakeholder group view.

tdTag is the fieldId in this case, but in other cases it can change, that is why I have passed one fieldId param.
Let me try this for other views, if it works, it should handle all view in a generic way

Reply:
@ganeshhubale , I tested this for stakeholder, and it works. You can try this on stakeholder group, in your upcoming PRs